operations/helm/charts/mimir-distributed/templates/ingester/ingester-statefulset.yaml

```diff
@@ -52,8 +52,17 @@ spec:
     metadata:
       labels:
         {{- include "mimir.podLabels" $args | nindent 8 }}
+        grafana.com/no-downscale: {{$rolloutZone.noDownscale}}
+        grafana.com/prepare-downscale: {{$rolloutZone.prepareDownscale}}
       annotations:
         {{- include "mimir.podAnnotations" $args | nindent 8 }}
+        {{- if $rolloutZone.prepareDownscale }}
+        grafana.com/prepare-downscale-http-path: ingester/prepare-shutdown
+        grafana.com/prepare-downscale-http-port: 80
```

Comment:
I don't remember if the default HTTP port for Mimir is changed in the helm chart. This might be `8080` these days.